common/generated_libs/network_protocol/network_protocol.pb.h

```diff
@@ -188,9 +194,9 @@ extern const pb_msgdesc_t Alarm_msg;
 
 /* Maximum encoded size of messages (where known) */
 #define GuiStatus_size                           140
-#define ControllerStatus_size                    167
+#define ControllerStatus_size                    177
 #define VentParams_size                          67
-#define SensorReadings_size                      15
+#define SensorReadings_size                      25
 #define Alarm_size                               13
```

Comment:
I'm not really familiar with this code, haven't spent any time in this file yet.  Are we really hard coding the sizes of structures?  Seems like something that's prone to error.

Reply:
This file is autogenerated by nanoproto from the `.proto` file.